glyph/src/ttfp.rs

```diff
@@ -181,6 +181,17 @@ impl FontVec {
 macro_rules! impl_font {
     ($font:ty) => {
         impl Font for $font {
+
+            #[inline]
+            fn as_slice(&self) -> &[u8] {
+                self.0.as_face_ref().raw_face().data
+            }
+
+            #[inline]
+            fn into_vec(&self) -> Vec<u8> {
+                Vec::from(self.0.as_face_ref().raw_face().data)
```

Comment:
into_vec should imply no cloning. If this clones users may as well use `as_slice` instead. So I don't think `FontArc` should implement `into_vec` as it can't do so.

Reply:
So perhaps just adding a `as_slice` method to `FontArc` is enough here?

Reply:
Makes sense, will remove